packages/block-editor/src/components/block-pattern-setup/index.js

```diff
@@ -145,7 +145,7 @@ const BlockPatternSetup = ( {
 	clientId,
 	blockName,
 	filterPatternsFn,
-	startBlankComponent,
+	startBlankComponent = null,
```

Comment:
I think we can remove the startBlankComponent, it is not being used anywhere, so I guess it is just something leftover from a chance and we missed its removal.

Reply:
Let's do this in a follow up.